README.md

```diff
@@ -43,6 +41,15 @@ As such, we recommend you install it in the following way.
    ```bash
    python tutorials/simple_example.py
    ```
+1. *(optional)* Check whether your local version of dockstring matches the dockstring dataset.
+   This is only necessary if you plan to mix docking scores from the dockstring dataset
+   with locally-computed scores, or if you want to compare results with the dockstring paper.
+   We have created a `pytest` test for this. To run it:
+   ```bash
+   conda install -c conda-forge pytest  # only if not installed already
+   python -m pytest tests/test_dataset_matching.py
+   ```
+   If the test passes then your local version of docktring matches the dataset! 🥳
```

Comment:
`If the test passes then your local version of dockstring matches the dataset EXACTLY!`